src/services/users/users.validations.ts

```diff
@@ -24,9 +20,63 @@ export const createUserSchema = Joi.object({
     .required(),
   password: Joi
     .string()
-    .required()
+    .required(),
+  title: Joi
+    .string()
+    .trim()
+    .min(1)
+    .max(100),
 });
 
+export const updateUserSchema = Joi.object({
+  firstName: Joi
+    .string()
+    .trim()
+    .min(1)
+    .max(100),
+  lastName: Joi
+    .string()
+    .trim()
+    .min(1)
+    .max(100),
+  email: Joi
+    .string()
+    .trim()
+    .email(),
+  password: Joi
+    .string(),
+  title: Joi
+    .string()
+    .trim()
+    .min(1)
+    .max(100),
+});
+
+export const userId = Joi.object({
+  id: Joi.string().uuid().required()
+});
+
+export const paramsSchema = Joi.object({
```

Comment:
Maybe better name is findFilters or getUsersFilters etc. ?

Reply:
changed ed2fcab9e7527ff7434ea9879bb56714970dc4f7